vllm/config.py

```diff
@@ -819,7 +821,20 @@ def maybe_create_spec_config(
                 window, if provided.
             ngram_prompt_lookup_min (Optional[int]): Min size of ngram token
                 window, if provided.
-
+            draft_token_sampling_method (Optional[str]): The sampling method
+                to use for accepting draft tokens. This can take two possible
+                values 'rejection_sampler' and 'typical_acceptance_sampler'
+                for RejectionSampler and TypicalAcceptanceSampler
+                respectively.
```

Comment:
Can we call this `spec_acceptance_routine` or `spec_sampling_method`? `draft_token_sampling_method` makes it seem this is what is used to sample only draft tokens.

Reply:
Renamed this to spec_decoding_acceptance_method. Replaced the name draft_token_sampling_method with draft_token_acceptance_method to avoid confusion.